lib/std/private/winutils.nim

```diff
@@ -0,0 +1,2 @@
+type
+  DWORD* = culong
```

Comment:
A DWORD is not defined in terms of C's typing system, a Windows DWORD is always 32 bits.

More importantly, move the declaration to where you need it instead of adding a file with 2 lines in it.

Reply:
> typedef unsigned long DWORD, *PDWORD

it is defined as `unsigned long` on 32/64 bit windows, I'll use uint32 instead of int32 which is defined in winlean.

Reply:
Ok.

Reply:
in future work we should have a module `wintypes.nim` and import it in winlean, io.nim (for windows) etc; it'd contain all the types from https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types that are needed in winlean, io;

> This type is declared in IntSafe.h as follows:
> typedef unsigned long DWORD;

so we should use:
```nim
type DWORD {.importc: "<IntSafe.h>".} = culong
```